README.md

```diff
@@ -1,135 +1,450 @@
+   
+   For example:
+
+   ```javascript
+   require("@nomicfoundation/hardhat-toolbox");
+   require("@nomicfoundation/hardhat-verify");
+   const path = require("path");
+   const fs = require("fs");
+   
+   // Read and parse the accounts file
+   const devnetAccountsPath = path.join(__dirname, 'devnet-accounts.json');
+   const devnetAccountsFile = fs.readFileSync(devnetAccountsPath, 'utf8');
+   const devnetAccounts = JSON.parse(devnetAccountsFile);
+   
+   // Validate account configuration
+   const requiredAccounts = 20;
+   if (devnetAccounts.accounts.length !== requiredAccounts) {
+     throw new Error(`Expected ${requiredAccounts} accounts in devnet-accounts.json, found ${devnetAccounts.accounts.length}`);
+   };
+   ```
+
+4. Copy and paste the code to configure network information from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project. 
+
+   For example:
+
+   ```javascript
+     defaultNetwork: "kadena_devnet0",
+     networks: {
+       kadena_devnet0: {
+         url: 'http://localhost:8545',
+         chainId: 1789,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 0,
+       },
+       kadena_devnet1: {
+         // url: 'http://localhost:8546',
+         url: 'http://localhost:8555',   
+         chainId: 1790,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 1,
+       },
+     },
+     ```
+
+1. (Optional) Copy and paste the code to configure etherscan settings from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project.
+   
+   For example:
+
+   ```javascript
+     defaultNetwork: "kadena_devnet0",
+     etherscan: {
+       apiKey: {
+         'kadena_devnet0': 'empty',
+         'kadena_devnet1': 'empty',
+       }, 
+       customChains: [
+         {
+           network: "kadena_devnet0",
+           chainId: 1789,
+           urls: {
+             apiURL: "http://localhost:8000/api",
+             browserURL: "http://localhost:8000"
+           }
+         },
+         {   
+           network: "kadena_devnet1",
+           chainId: 1790,
+           urls: {
+             apiURL: "http://localhost:8001/api",
+             browserURL: "http://localhost:8001"
+           }
+         },
+       ]
+     },
+     ```
+1. Save your changes and close the `hardhat.config.js` file.
+
+### Compiling and testing integration
+
+After configuring your Hardhat project to use the Chainweb EVM development network, you can compile, test, and deploy the project using standard `hardhat` commands.
+For example, compile the project by running the following command:
 
 ```sh
-./network devnet pull
+npx hardhat compile
 ```
 
-Start the network:
+Run unit tests using the standard `hardhat test` command:
 
 ```sh
-./network devnet start
+cd solidity 
```

Comment:
Line 217 explains that the `solidity` directory is an exmaple hardhat project. None of the other command examples show a `cd` to the `solidity` directory first, so I think this line can be deleted.